### PR TITLE
Flow dir cond

### PR DESCRIPTION
**Pull request by pkdash**

### CODE REVIEW

**Comment by pkdash** [comment]
@dtarb Can you review this? Let me know if it can be merged.

File: Taudem5PCVS2010/FlowdirConditioning/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
+========================================================================
```
Comment (dtarb) [inline Taudem5PCVS2010/FlowdirConditioning/ReadMe.txt]: This file appears to have been automatically generated.  I prefer not to have it.  See if we can configure project without it.

File: Taudem5PCVS2010/FlowdirConditioning/stdafx.cpp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// stdafx.cpp : source file that includes just the standard includes
```
Comment (dtarb) [inline Taudem5PCVS2010/FlowdirConditioning/stdafx.cpp]: This file appears to have been automatically generated. I prefer not to have it. See if we can configure project without it.

File: Taudem5PCVS2010/FlowdirConditioning/stdafx.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// stdafx.h : include file for standard system include files,
```
Comment (dtarb) [inline Taudem5PCVS2010/FlowdirConditioning/stdafx.h]: This file appears to have been automatically generated. I prefer not to have it. See if we can configure project without it.

File: Taudem5PCVS2010/FlowdirConditioning/targetver.h [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#pragma once
```
Comment (dtarb) [inline Taudem5PCVS2010/FlowdirConditioning/targetver.h]: This file appears to have been automatically generated. I prefer not to have it. See if we can configure project without it.

File: Taudem5PCVS2015/Taudem5PC.sln [hunk 5]
```diff
@@ -63,6 +63,7 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ogrtest", "ogrtest\ogrtest.
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SinmapSI", "SinmapSI\SinmapSI.vcxproj", "{4C94C630-FAFC-4EC9-8639-336E6FD7A85C}"
```
Comment (dtarb) [inline Taudem5PCVS2015/Taudem5PC.sln]: does this follow the new pattern developed for project/solution files in develop branch.

File: src/tiffIO.cpp [hunk 6]
```diff
@@ -160,12 +160,21 @@ tiffIO::tiffIO(char *fname, DATA_TYPE newtype) {
 	if (datatype == SHORT_TYPE) {
```
Comment (dtarb) [inline src/tiffIO.cpp]: Flowdircond should not be changing tiffIOI.cpp.  It should be using what is the latest from develop branch.

Comment (dtarb) [inline src/tiffIO.cpp]: These may be fixes that are really a separate issue.  If we are OK that this is the case, and tests all work, then OK to merge.

File: Taudem5PCVS2010/FlowdirConditioning/FlowdirConditioning.vcxproj [hunk 7]
```diff
@@ -0,0 +1,150 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pkdash) [inline Taudem5PCVS2010/FlowdirConditioning/FlowdirConditioning.vcxproj]: @dtarb Is this project needed? It seems this branch has 2 new projects - FlowdirCond and FlowdirConditioning

**Comment by pkdash** [comment]
@dtarb I have now updated and cleaned this branch.

**Pull request merged**